packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -7,17 +7,32 @@ import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
+import { HdsCopyButtonSizeValues } from './types.ts';
+import type { HdsCopyButtonSizes } from './types.ts';
+import type { HdsButtonSignature } from '../../button/';
+import type { ModifierSignature } from '../../../../modifiers/hds-clipboard.ts';
 
-export const DEFAULT_SIZE = 'medium';
-export const SIZES = ['small', 'medium'];
+export const DEFAULT_SIZE = HdsCopyButtonSizeValues.Medium;
+export const SIZES: string[] = Object.values(HdsCopyButtonSizeValues);
 export const DEFAULT_ICON = 'clipboard-copy';
 export const SUCCESS_ICON = 'clipboard-checked';
 export const ERROR_ICON = 'clipboard-x';
 export const DEFAULT_STATUS = 'idle';
 
-export default class HdsCopyButtonComponent extends Component {
+export interface HdsCopyButtonSignature {
+  Args: HdsButtonSignature['Args'] & {
+    size?: HdsCopyButtonSizes;
+    textToCopy?: ModifierSignature['Args']['Named']['text'];
```

Comment:
@alex-ju I suggested Kristin to do this. Do you agree with this approach? Other/better ideas?

Reply:
considering the two (the component and the modifier) share a similar structure for the signature, yes I think this makes sense. alternatively, we could create a shared interface and update both `HdsCopyButtonSignature ` and `HdsClipboardModifierSignature` to use it, but to me they don't seem like siblings as the CopyButton component implements the modifier, so this format works well